build_tools/benchmarks/common/linux_device_utils_test.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import unittest
+
+from unittest.mock import patch
```

Comment:
I think better to keep this with some kind of namespacing. When I look at a function call to `patch` it's not immediately obvious what it does. `mock.patch` on the other hand, is much clearer. We're pretty loose about the Python style guide, but this particular guidance is good, I think. https://google.github.io/styleguide/pyguide.html#s2.2-imports

```suggestion
from unittest import mock
```

Reply:
Done. Thx